meta-lmp-base/recipes-multimedia/alsa/alsa-utils_%.bbappend

```diff
@@ -0,0 +1,10 @@
+# Try to restore /usr/lib/alsa/asound.state if /var/lib/alsa/asound.state
+# is unavailable (it happens on ostree-based distro).
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
+
+EXTRA_OECONF:append = " --with-init-asound-state-dir=${prefix}/lib/alsa \
+                        --with-asound-state-dir=${localstatedir}/lib/alsa "
+EXTRA_OEMAKE:append = " SYS_ASOUND_DIR=${localstatedir}/lib/alsa"
+
+SRC_URI += " file://0001-alsactl-add-fallback-for-restoring-from-asound.state.patch "
```

Comment:
With += you don't need empty space before/after the line.

Reply:
Fxed